File: app/src/main/java/ru/ls/donkitchen/rest/model/response/ReviewListResult.java
package ru.ls.donkitchen.rest.model.response;

import com.google.gson.annotations.SerializedName;

import org.joda.time.DateTime;

import java.util.List;

/**
 * @author Lord (Kuleshov M.V.)
 * @since 27.11.15
 */
public class ReviewListResult {

	/**
	 * id : 5
	 * receiptId : 1
	 * date : 2015-11-27 13:57:59
	 * rating : 3
	 * userName : Максим
	 * comments : Отличный рецепт! Всем рекоммендую!
	 */

	@SerializedName("reviews")
	public List<ReviewItem> reviews;

	public static class ReviewItem {
		@SerializedName("id")
		public int id;
		@SerializedName("receiptId")
		public int receiptId;
		@SerializedName("date")
		public DateTime date;
		@SerializedName("rating")
		public int rating;
		@SerializedName("userName")
		public String userName;
		@SerializedName("comments")
		public String comments;
	}
}

File: data/src/main/kotlin/ru/ls/donkitchen/data/rest/response/ReviewListResult.kt
package ru.ls.donkitchen.data.rest.response

import com.google.gson.annotations.SerializedName
import java.util.*

/**
 *
 *
 * @author Lord (Kuleshov M.V.)
 * @since 16.10.16
 */
class ReviewListResult {
    /**
     * id : 5
     * receiptId : 1
     * date : 2015-11-27 13:57:59
     * rating : 3
     * userName : Максим
     * comments : Отличный рецепт! Всем рекоммендую!
     */

    @SerializedName("reviews")
    var reviews: List<ReviewItem>? = null

    class ReviewItem {
        @SerializedName("id")
        var id: Int = 0
        @SerializedName("receiptId")
        var receiptId: Int = 0
        @SerializedName("date")
        var date: Date? = null
        @SerializedName("rating")
        var rating: Int = 0
        @SerializedName("userName")
        var userName: String? = null
        @SerializedName("comments")
        var comments: String? = null
    }
}